### ISSUE TITLE
Verify Twitter Grant Issue

### ISSUE THREAD

**Issue by WilfredTA** [gitcoinco/web#8026]
**Who**

Pictosis is experiencing trouble verifying the grant on Twitter account.

**What**
Here is the grant page: [Pictosis Gitcoin Grant](https://gitcoin.co/grants/1663/pictosis-the-open-web-for-digital-goods-creative-)

I can create the tweet on twitter, but it is not detected on Gitcoin

Please advise. [here is the link to the tweet](https://twitter.com/pictosis/status/1334510395411804163?s=20)

**Why**

When people are exploring grants, it seems that non-verified accounts get lower priority / filtered out and it undermines our credibility

**Comment by developerfred** [comment]
@WilfredTA look https://github.com/gitcoinco/web/issues/8009

**Comment by developerfred** [comment]
@WilfredTA The message needs to be the one that the Gitcoin grants page generates for you
ex: https://twitter.com/realcodecamp/status/1334572427637125122?s=20

**Comment by WilfredTA** [comment]
@developerfred  The message that the Gitcoin grants page generates for me does not have a link to the gitcoin grant page. This is the message generated by Gitcoin grants page: https://twitter.com/pictosis/status/1334503121951862784?s=20

Whereas the link I posted above was my attempt to modify it to add in the link to the grant itself.

**Comment by WilfredTA** [comment]
@developerfred Any idea on a way to resolve this?

**Comment by developerfred** [comment]
@WilfredTA modifying the message will not work, even the emojis of the message are relevant to validate. the Message must be identical.

**Comment by WilfredTA** [comment]
> @WilfredTA modifying the message will not work, even the emojis of the message are relevant to validate. the Message must be identical.

The tweet I linked to above (and here): https://twitter.com/pictosis/status/1334503121951862784?s=20 

is the *unmodified*, identical tweet that is created by the gitcoin page. Whenever I click the verification button to generate the tweet for me, that is what it produces.

**Comment by developerfred** [comment]
@WilfredTA I just checked out two other grants I have and it was super smooth. It came with the link and the emojis.

**Comment by WilfredTA** [comment]
@developerfred here is a link demonstrating what is happening: https://streamable.com/hhtdcg

**Comment by developerfred** [comment]
@WilfredTA Realmente é um bug,  preciso analisar agora se é por causa da url do grants https://gitcoin.co/grants/1663/pictosis-the-open-web-for-digital-goods-creative- [80 charecters] 

at some point it is breaking, twitter has to fit the message to the url of the grants, if we use a shortener you can prevent these errors.

**Comment by WilfredTA** [comment]
@developerfred Ah okay makes sense. In the meantime, what can we do in order to get verified so that our project isn't as filtered out by the curation of grants on the home page?

Is there a way I could potentially change the name? Then, of course, we'd have to make sure that everywhere that references the URL has also been updated..

**Comment by thelostone-mc** [comment]
@WilfredTA I've verified you manually , so that should unblock you
meanwhile -> we'll figure out a fix for this 🙌

**Comment by octavioamu** [comment]
ok we figure out with @chibie what is causing this bug, is not the length of the text is the encoding of "&" . Chibie is setting up a fix.
